server/sonar-server/src/main/java/org/sonar/server/permission/index/PermissionIndexer.java

```diff
@@ -56,11 +60,12 @@
  *   <li>delete project orphans from index</li>
  * </ul>
  */
-public class PermissionIndexer implements ProjectIndexer, Startable {
+public class PermissionIndexer implements ProjectIndexer, Startable, StartupIndexer {
 
   @VisibleForTesting
   static final int MAX_BATCH_SIZE = 1000;
 
+  private static final Logger LOG = Loggers.get(PermissionIndexer.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "LOG" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)